packages/riipen-ui/src/components/Radio.test.jsx

```diff
@@ -0,0 +1,152 @@
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Radio />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Radio().type.defaultProps;
+
+      const wrapper = mount(<Radio />);
+
+      expect(wrapper.find("Radio").props().checked).toEqual(
+        defaultProps.checked
+      );
+      expect(wrapper.find("Radio").props().disabled).toEqual(
+        defaultProps.disabled
+      );
+    });
+  });
+
+  describe("checked prop", () => {
+    it("sets valid custom checked", () => {
+      const checked = true;
+
+      const wrapper = mount(<Radio checked={checked} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .find("input")
+          .props().checked
+      ).toEqual(checked);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<Radio classes={classes} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-radio"];
+
+      const wrapper = mount(<Radio />);
+
+      expect(wrapper.find("Radio").props().classes).toEqual(classes.sort());
+    });
+  });
+
+  describe("color prop", () => {
+    it("sets valid custom  color", () => {
+      const color = "default";
+
+      const wrapper = mount(<Radio color={color} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .find("Typography")
+          .childAt(0)
+          .hasClass(color)
+      ).toBeTruthy();
+    });
+
+    it("gives an error when given an invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+      const color = "purple";
+
+      mount(<Radio color={color} />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("disabled prop", () => {
+    it("renders radio as disabled when disabled prop is true", () => {
+      const disabled = true;
+
+      const wrapper = mount(<Radio disabled={disabled} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .find("span")
+          .hasClass("disabled")
+      ).toBeTruthy();
+    });
+
+    it("renders radio with color when disabled prop is false", () => {
+      const color = "primary";
+
+      const wrapper = mount(<Radio color={color} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .find("span")
+          .hasClass(color)
+      ).toBeTruthy();
+      expect(
+        wrapper
+          .find("Radio")
+          .find("span")
+          .hasClass("disabled")
+      ).toBeFalsy();
+    });
+  });
```

Comment:
I think you meant to check `const disabled = false;`, not color.

Reply:
disabled is set to false by default, so with this test I was trying to target line 90 in the component file.

```
 className={clsx("checkmark", disabled ? "disabled" : null, color)}
```